### PR TITLE
- Fixed skip button loop again

### PR DESCRIPTION
**Pull request by BrandenB**

### CODE REVIEW

File: resources/web/ytplayer/js/ytPlayer.js [hunk 1]
```diff
@@ -3,7 +3,7 @@
  * ------------
  * Interface for the YouTube Player for PhantomBot.
  */
-var DEBUG_MODE = false;
+var DEBUG_MODE = true;
```
Comment (ddugovic) [inline resources/web/ytplayer/js/ytPlayer.js:6]: I wonder, is this change intentional?

**Comment by BrandenB** [comment]
It was reverted.
